dev-libs/libjwt/metadata.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person" proxied="yes">
+		<email>nic@boet.cc</email>
+		<name>Nic Boet</name>
+	</maintainer>
+	<maintainer type="person" proxied="yes">
+		<email>jaco@uls.co.za</email>
+		<name>Jaco Kroon</name>
+	</maintainer>
+	<maintainer type="person" proxied="yes">
+		<email>dek.devel@baisenvar.info</email>
```

Comment:
Has this person agreed to maintain it as well?

Reply:
Yes.  @dekeonus please advise if I'm wrong.

Reply:
@jkroonza you are correct